docs/UserGuide.md

```diff
@@ -284,22 +334,20 @@ _Details coming soon ..._
 
 ## Command summary
 
-Action | Format, Examples
+Command | Format, Examples
 --------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​` <br> e.g. `add n/John Doe p/91234567 e/johndoe@example.com t/Groom's Side`
+**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL [t/TAG]…​` <br> e.g. `add n/John Doe p/91234567 e/johndoe@example.com t/groom's side`
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [t/TAG]…​`<br> e.g.`edit 2 n/James Lee e/jameslee@example.com`
 **Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g. `find James Jake`
 **List** | `list`
-**NewTag** | `newtag TAG_NAME` <br> e.g. `newtag Bride's Side`
-**DeleteTag** | `deletetag t/TAG_NAME...` <br> e.g. `deletetag t/Bride's Side`
-**Tag** | `tag INDEX t/TAG` <br> e.g. `tag 2 t/Groom's Side`
-**Untag** | `untag INDEX t/TAG` <br> e.g. `untag 1 t/Bride's Side`
-**RSVP** | `rsvp INDEX`
-**UnRSVP** | `unrsvp INDEX`
-**RSVPList** | `rsvplist`
-**NotRSVPList** | `notrsvplist`
+**NewTag** | `newtag TAG` <br> e.g. `newtag bride's side`
```

Comment:
`newtag` requries the prefix t/ when creating new tags. Can also include the `...` to show that multiple tags can be created at once